### ISSUE TITLE
Personal-Level instance blocks?

### ISSUE THREAD

**Issue by golbette** [Gargron/mastodon#245]
We are able to block (ie, effectively 'mute') users individually, and I noticed there is an issue from a very long time ago that mentioned that this can also be done for entire domains/'instances,' and is already done on the, erm, 'federation' level? (Basically, the whole Mastodon instance can have entire other instances muted/blocked for everyone.) Would it be possible to add a way to block instances or domains users don't want to see on an individual level?

(E.g., say there exists an instance called, I dunno, 'shitworld.ru' that isn't blocked by mastodon.social, and I personally don't want to see 'shitworld.ru', so I hit a button that 'blocks'/mutes all posts from 'shitworld.ru' on my timelines. Other people can still see shitworld.ru posts, and obviously nothing would stop shitworld.ru from interacting with me, but I wouldn't have to see them on my TLs.) 

(Apologies if this is a duplicate, I couldn't find anything else on the issue w/ search.)

**Comment by hoodiek** [comment]
this is addressed in this thread: https://github.com/Gargron/mastodon/issues/231 but is definitely something worth existing on a user level, which is a request in said thread. maybe give it a thumbs up or whatever

**Comment by hoodiek** [comment]
i worded this badly: what i meant to say was i adding it to that thread, and thus trying to merge them. i think it's important to organize this stuff easy for @Gargron to lessen that workload. please talk there about such things if my wording is inaccurate. thanks for listening

**Comment by swaldie** [comment]
Since this is added to #231, I'm going to close this issue and reference it there.  Thank you for mentioning this, @hoodiek
